brownie/network/contract.py

```diff
@@ -55,59 +63,63 @@ class ContractContainer(_ContractBase):
         signatures: Dictionary of {'function name': "bytes4 signature"}
         topics: Dictionary of {'event name': "bytes32 topic"}'''
 
-    def __init__(self, project, build):
+    def __init__(self, project: Any, build: Dict) -> None:
         self.tx = None
         self.bytecode = build['bytecode']
-        self._contracts = []
+        self._contracts: List = []
         super().__init__(project, build, build['contractName'], build['abi'])
         self.deploy = ContractConstructor(self, self._name)
         rpc._revert_register(self)
 
-    def __iter__(self):
+    def __iter__(self) -> Iterable:
         return iter(self._contracts)
 
-    def __getitem__(self, i):
+    def __getitem__(self, i: Any) -> Any:
         return self._contracts[i]
 
-    def __delitem__(self, key):
+    def __delitem__(self, key: Any) -> None:
         item = self._contracts[key]
-        history._remove_contract(item)
+        _remove_contract(item)
         self._contracts.remove(item)
 
-    def __len__(self):
+    def __len__(self) -> int:
         return len(self._contracts)
 
-    def __repr__(self):
+    def __repr__(self) -> str:
         return str(self._contracts)
 
-    def _reset(self):
+    def _reset(self) -> None:
         for contract in self._contracts:
-            history._remove_contract(contract)
+            _remove_contract(contract)
             contract._reverted = True
         self._contracts.clear()
 
-    def _revert(self, height):
+    def _revert(self, height: int) -> None:
         reverted = [
             i for i in self._contracts if
             (i.tx and i.tx.block_number > height) or
             len(web3.eth.getCode(i.address).hex()) <= 4
         ]
         for contract in reverted:
-            history._remove_contract(contract)
+            _remove_contract(contract)
             self._contracts.remove(contract)
             contract._reverted = True
 
-    def remove(self, contract):
+    def remove(self, contract: str) -> None:
```

Comment:
`contract` will be a `Contract` instance, but maybe this is to avoid some mypy confusion?